### DISCUSSION TITLE
How far away is unified inbox?

### DISCUSSION THREAD

**Discussion by banch7** [bkaankose/Wino-Mail#189] [General]
Love the app ... I installed it a few days ago and have tweaked it well and love what you've done.

I have read back and saw you mentioned a unified inbox is coming ... just wanted to check if it's far off? I have 6 accounts total so am willing to pay for unlimited accounts, but the deal breaker right now is a unified inbox as it's much easier to sort with 6 email accounts.

Appreciate all your work

**Comment by banch7** [comment]
Also, another quick question ... any possibility of adding "Minimize to Tray" option. It would be great when X'ing out of the app, that it minimizes to the tray but still runs in the background ... out of sight.

**Comment by Eyji** [comment]
Unified inbox is the number 1 feature missing in the new Outlook that killed the previous Mail app. 
Would be greatly appreciated to have that feature in this otherwise beautiful app

**Comment by bkaankose** [comment]
Unified inbox is released for Beta version. I'm trying to release it to the Store, but couple internal bugs are preventing me to do it :)

It's not far, maybe a week or couple days. As long as I fixed 1 critical issue regarding Outlook synchronization I will send it to the Microsoft Store.

**Comment by bkaankose** [comment]
System tray thing is not possible since underlying platform does not support it without requiring all your computer access, which I don't want to ask for people for during installation.

Startup performance of Wino is pretty good though. App doesn't need to stay running all the time. Background synchronization will kick in every 15 minutes for new incoming mails anyways, without even launching it.

**Comment by Eyji** [comment]
Regarding your previous statement: does this mean the application continues to operate even when closed? I'm skeptical because you mentioned it lacks support for minimizing to the tray, yet that was my interpretation.

**Reply by bkaankose** [reply]
> Regarding your last statement: does that mean it continues to run even when it's closed?

Yes. Synchronization can run in the background.

**Comment by bkaankose** [comment]
Closing the discussion since unified inbox feature went live with version 1.7.2.
